### PR TITLE
Add procs/short cooldowns section to mw guide

### PR DESCRIPTION
**Pull request by trevorm4**
Added EF to core spells section (because it is a core spell) and added chi burst, sheiluns, mana tea, vivacious vivification and ancient teachings to the short cooldowns section. Added avg vivify cleaves to rem subsection and marked guide as default

image
image
image
image

### CODE REVIEW

File: src/analysis/retail/monk/mistweaver/Guide.tsx [hunk 1]
```diff
@@ -47,6 +59,64 @@ function HotGraphSubsection({ modules, events, info }: GuideProps<typeof CombatL
...
+        <SpellLink id={SPELLS.VIVIFY} /> during a{' '}
+        <SpellLink id={TALENTS_MONK.RENEWING_MIST_TALENT} /> peak (8-10+ active HoTs) or before
+        casting{' '}
+        {info.combatant.hasTalent(TALENTS_MONK.INVOKE_CHI_JI_THE_RED_CRANE_TALENT) ? (
+          <SpellLink id={TALENTS_MONK.INVOKE_CHI_JI_THE_RED_CRANE_TALENT} />
+        ) : (
+          <SpellLink id={TALENTS_MONK.INVOKE_YULON_THE_JADE_SERPENT_TALENT} />
+        )}
+        .
+      </p>
```
Comment (vohrr) [inline src/analysis/retail/monk/mistweaver/Guide.tsx:91]: **Cast with as little as one stack before revival

File: src/analysis/retail/monk/mistweaver/modules/spells/SheilunsGiftCloudTracker.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import SPELLS from 'common/SPELLS';
+import { Options } from 'parser/core/Analyzer';
+import BuffStackTracker from 'parser/shared/modules/BuffStackTracker';
+
+export default class SheilunsGiftCloudTracker extends BuffStackTracker {
+  static trackPets = false;
+  static trackedBuff = SPELLS.SHEILUN_CLOUD_BUFF;
```
Comment (vohrr) [inline src/analysis/retail/monk/mistweaver/modules/spells/SheilunsGiftCloudTracker.tsx:7]: Will this pick up SG cloud buffs from all mistweavers in a raid group?

File: src/analysis/retail/monk/mistweaver/modules/spells/VivaciousVivify.tsx [hunk 3]
```diff
@@ -2,26 +2,33 @@ import { t } from '@lingui/macro';
...
   };
-  protected invokeChiji!: InvokeChiJi;
+  protected baseCelestial!: BaseCelestialAnalyzer;
   protected renewingMist!: RenewingMist;
   protected vivify!: Vivify;
   currentRenewingMists: number = 0;
   totalCasts: number = 0;
   totalHealed: number = 0;
   wastedApplications: number = 0;
+  unusableUptimes: Uptime[] = []; // a wasted window is when we have buff and good rem count
```
Comment (vohrr) [inline src/analysis/retail/monk/mistweaver/modules/spells/VivaciousVivify.tsx:31]: Outside of celestial windows

**Review by vohrr — APPROVED** [review]
Lgtm, couple small notes on the guide subsection wording

File: src/analysis/retail/monk/mistweaver/modules/spells/SheilunsGiftCloudTracker.tsx (hunk 2 above)
Comment (trevorm4) [reply src/analysis/retail/monk/mistweaver/modules/spells/SheilunsGiftCloudTracker.tsx:7]: Nope, it's just for current player

**Comment by vohrr** [comment]
The comments are minor on this and don't break analysis so I'm going to merge this to get going on pt 4

**Pull request merged**